core/src/main/scala/org/apache/spark/Dependency.scala

```diff
@@ -122,6 +119,14 @@ class ShuffleDependency[K: ClassTag, V: ClassTag, C: ClassTag](
    */
   private[this] var _shuffleMergedFinalized: Boolean = false
 
+  /**
+   * shuffleSequenceId is used to give temporal ordering to the executions of a ShuffleDependency.
+   * This is required in order to handle indeterministic stage retries for push-based shuffle.
+   */
+  private[this] var nextShuffleSequenceId: Int = 0
```

Comment:
`nextShuffleSequenceId` -> `_shuffleSequenceId `

Reply:
I don't see anywhere that increase it. Is it in progress?